src/ts/proposal.ts

```diff
@@ -85,10 +97,11 @@ export async function generateProposal(
     ethers,
   );
 
+  const totalSupply = BigNumber.from(10).pow(3 * 3 + metadata.real.decimals);
   const realTokenDeployParams: RealTokenDeployParams = {
     cowDao,
-    initialTokenHolder: cowDao,
-    totalSupply: BigNumber.from(10).pow(3 * 3 + metadata.real.decimals),
+    initialTokenHolder: settings.gnosisDao,
```

Comment:
I remember @josojo mentioning something like this in the meeting yesterday morning. This requires a minor code change to #65 WRT computing the init code hash - so don't mine any complicated vanity addresses for the COW token quite yet.

It is safe to do for the two Safes though.